templates/controllers/modals/editorDecision/form/promoteForm.tpl

```diff
@@ -27,41 +27,53 @@
 	<input type="hidden" name="decision" value="{$decision|escape}" />
 	<input type="hidden" name="reviewRoundId" value="{$reviewRoundId|escape}" />
 
-	{if array_key_exists('help', $decisionData)}
-		<p>{translate key=$decisionData.help}</p>
-	{/if}
-
-	{fbvFormSection title="user.role.author_s" for="authorName" size=$fbvStyles.size.MEDIUM}
-		{fbvElement type="text" id="authorName" name="authorName" value=$authorName disabled=true}
-	{/fbvFormSection}
-
-	{fbvFormSection title="editor.review.personalMessageToAuthor" for="personalMessage"}
-		{fbvElement type="textarea" name="personalMessage" id="personalMessage" value=$personalMessage rich=true variables=$allowedVariables variablesType=$allowedVariablesType}
-	{/fbvFormSection}
+	<div id="promoteForm-step1">
+		{if array_key_exists('help', $decisionData)}
+			<p>{translate key=$decisionData.help}</p>
+		{/if}
 
-	{* Button to add reviews to the email automatically *}
-	{if $reviewsAvailable}
-		{fbvFormSection}
-			<a id="importPeerReviews" href="#" class="pkp_button">
-				<span class="fa fa-plus" aria-hidden="true"></span>
-				{translate key="submission.comments.addReviews"}
-			</a>
+		{capture assign="sendEmailLabel"}{translate key="editor.submissionReview.sendEmail" authorName=$authorName}{/capture}
+		{if $skipEmail}
+			{assign var="skipEmailSkip" value=true}
+		{else}
+			{assign var="skipEmailSend" value=true}
+		{/if}
```

Comment:
I believe `$skipEmail` is either `null` or `string`, and the radio button's `checked` attribute expects something which can resolve to a `boolean` (eg - `{if $checked}`. I don't think that I can pass an operation, like `checked=!$skipEmail` or `checked=empty($skipEmail)` to the radio button form element, can I? How would you do this without the variables?

Reply:
@NateWr, I think we actually do not need this at all -- the $skipEmail is never set when the template is called, that variable is just needed when the form is saved/executed, correct? Thus I believe we could only say checked=true for the first radio button, in order for it to be checked per default, and remove those if-else statement and variables before. What do you think?

Reply:
Can the form return an error and if so is the variable passed back into the form? If we never need to persist the user's selection when re-rendering the form, then you're right this can be removed.

Reply:
Yes I think the user's selection is then saved in the samenamed field so it is available when the form is re-rendered, which is currently never the case, I think...

Reply:
So does that mean it should be removed? If so, can you file an issue?